config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,3 @@
+require 'syslog/logger'
+
+Sidekiq::Logging.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new('joegattnet_v3_sidekiq'))
```

Comment:
Line is too long. [103/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.